lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -52,24 +54,28 @@ def __init__(
     ):
         super().__init__()
         self.repo_id = repo_id
-        self.version = version
         self.root = root
         self.split = split
         self.image_transforms = image_transforms
         self.delta_timestamps = delta_timestamps
         # load data from hub or locally when root is provided
         # TODO(rcadene, aliberts): implement faster transfer
         # https://huggingface.co/docs/huggingface_hub/en/guides/download#faster-downloads
-        self.hf_dataset = load_hf_dataset(repo_id, version, root, split)
+        if root is not None:
+            self.version = "local"
```

Comment:
Good initiative! it makes me think that instead of using "local", we should probably add the CODEBASE_VERSION in the meta data and use this instead.
We want to have this backward compatibility feature on local datasets as well.

Could you add it for this PR?